platform/cog-platform-fdo.c

```diff
@@ -257,15 +267,20 @@ configure_surface_geometry (int32_t width, int32_t height)
 static void
 resize_window (void)
 {
+    int32_t pixel_width = win_data.width * wl_data.current_output.scale;
+    int32_t pixel_height = win_data.height * wl_data.current_output.scale;
+
     if (win_data.egl_window)
         wl_egl_window_resize (win_data.egl_window,
-			      win_data.width,
-			      win_data.height,
+			      pixel_width,
+			      pixel_height,
 			      0, 0);
 
     wpe_view_backend_dispatch_set_size (wpe_view_data.backend,
                                         win_data.width,
                                         win_data.height);
+    printf ("Resized EGL buffer to: (%u, %u) @%ix\n",
+            pixel_width, pixel_height, wl_data.current_output.scale);
```

Comment:
There are a few situations in which `printf` is being used where it would be much better to use [g_debug](https://developer.gnome.org/glib/stable/glib-Message-Logging.html#g-debug), or [g_message](https://developer.gnome.org/glib/stable/glib-Message-Logging.html#g-message) for things which need to be always logged.

(I am aware that the existing code is already using `printf`, but see #1—it is better  to avoid introducing new uses.)